css/style.css

```diff
@@ -0,0 +1,1003 @@
+  background: #f0f1ec;
+  padding: 20px;
+  max-width: 100px;
+  vertical-align: middle;
+} 
+.listing-style-two h4 {
+  text-align: left;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  color: #3a3a3a;
+}
+.listing-style-two {
+  flex-wrap: wrap;
+}
+.listing-style-two li {
+  flex: 0 1 33.33%;
+  padding: 20px;
+}
+.listing-style-two li img {
+  text-align: left;
+}
+.listing-style-two li:hover {
+  background-color: #fff;
+  border-radius: 12px;
+}
+.listing-style-three li {
+  flex: 33.33%;
+  padding: 20px;
+}
+.listing-style-three {
+  flex-wrap: wrap;
+} 
+.listing-style-three li img {
+  width: 100%;
+  object-fit: cover;
+  height: 100%;
+  vertical-align: middle;
+  border-radius: 5%;
+}
+.flex-box .slider-info img {
+  border-radius: 50%;
+  background: #709d61;
+  padding: 10px;
+  max-width: 50px;
+  height: 50px;
+  margin-bottom: 8px;
+}
+.block .flex-box img {
+  max-width: 300px;
+}
+.slider-info h4 {
+  text-align: left;
+  font: 20px "Manrope-Bold";
+  color: #000000;
+}
+.letter-box {
+  background-color: #f1f2ee;
+  border-radius: 10px;
+  padding: 50px 15px;
+}
+.block-center {
+  text-align: center;
+}
+.block .testimonial {
+  justify-content: center;
+}
+.mail-box {
+  border-radius: 10px;
+  padding: 10px;
+  max-width: 550px;
+  margin: 0 auto;
+  background: #fff;
+  display: flex;
+  justify-content: space-between;
+}
+.letter-box .mail-box
+input[type="email"] {
+  border: 0;
+  outline: none;
+  background: transparent;
+  width: 100%;
+  padding: 15px 20px;
+  font: 18px "Manrope-Bold";
+}
+.letter-box .mail-box 
+input[type="button"] {
+  font: 20px "Manrope-Bold";
+}
+
+/*................................................
+    7.FOOTER SECTION
+.................................................*/
+#footer .flex-box > li {
+  flex: 0 1 25%;
+  padding: 0 20px;
+}
+#footer {
+  padding: 20px;
+  border-top: 20px solid #f7f6f4;
+  border-bottom: 20px solid #f7f6f4;
+  vertical-align: middle;
+}
+#footer h4 {
+  text-align: left;
+  color: #3a3a3a;
+  margin-bottom: 10px;
+}
+#footer .footer-logo  {
+  padding: 0;
+  text-align: left;
+  margin-bottom: 10px;
+} 
+#footer .flex-box .useful-links li a {
+  text-align: left;
+  color: #656565;
+  line-height: 0.2;
+}
+.contact-list .day h5 {
+  font-size: 14px;
+}
+hr {
+  border: 1px solid #eee;
+}
+#footer .copyright h4 {
+  font: 14px "Manrope-Medium";
+  padding: 20px;
+  text-align: center;
+}
+
+/*................................................
+    8.MEDIA QUERIES
+.................................................*/
+@media (min-width: 991px) {
+  .desktop-only {
+    display: block;
+  }
+  .mobile-only {
+    display: none;
+  }
+  .nav-icon {
+    display: none;
+  }
+  .block-panel-left {
+    padding-right: 40px;
+  }
+  .block h3 {
+    color: #000;
+    font: 40px "Manrope-Bold";
+    margin-bottom: 30px;
+    text-align: center;
+  }
+  .listing-style-one li,
+  .listing-style-two li {
+    flex: 0 1 33.33%;
+  }
+  .listing-style-two li {
```

Comment:
If you are applying grid property then no need of flex, make sure to style with either flex or grid.